### DISCUSSION TITLE
How to identify parked domains?

### DISCUSSION THREAD

**Discussion by CodeAlDente** [Bon-Appetit/porn-domains#47] [Ideas]
To identify domains that are just parked and not really being used, we can look at their DNS records, especially the nameservers. Here's a list of some nameservers that are used for domain parking services. Can you guys help out and add more to the list? The final result will be used later on in background processes to maintain the list.

**Moved the list of nameservers used for parked/for sale domains to a dedicated Gist:**
https://gist.github.com/CodeAlDente/33aeb7ff369e7ecd2d52abd0d0ee7d59

**Comment by CodeAlDente** [comment]
**Some thoughts on this:**

1) For parked domains, ads could contain adult content either due to the domain's previous content or because of the domain name itself.

**Comment by spirillen** [comment]
The full list of DNS servers used exclusively for domain piracy, hijacking are way longer, than those few listed above.

You can at any time, see my list of DNS server exclusively used for this here. And the approach described in https://github.com/Bon-Appetit/porn-domains/discussions/6#discussioncomment-11720003, is a good way to do it.

https://github.com/mypdns/matrix/blob/master/source/pirated/wildcard.rpz-nsdname.csv

> For parked domains, ads could contain adult content either due to the domain's previous content or because of the domain name itself

Thru, but they are primarily based on your [PII](https://kb.mypdns.org/articles/MTX-A-79) identity, as they are served by the Ad and Spyware Company Alphabet (Google), I have chosen to disregard this, as I assume most people spending time assigning a NSFW filter, also ads a variety of Ads and tracking filters to their network

**Reply by CodeAlDente** [reply]
Sure, the list is just an example of few nameservers and there might be much more. The ones you've mentioned in your CSV file could be added later on. Thanks for sharing.

**Reply by spirillen** [reply]
You're welcome, and I hope you return the favor when you find more DNS server, exclusively used to host pirated domains

**Reply by CodeAlDente** [reply]
After testing and experimenting with different approaches, I've concluded that using the full nameserver name yields more accurate and reliable results, particularly when it comes to ensuring precision and avoiding unintended matches. When we use the **exact nameserver name** (e.g., `ns1.sedoparking.com`), we can ensure that we’re specifically targeting the correct nameserver, which minimizes the risk of false positives.

**Comment by CodeAlDente** [comment]
I’m closing this discussion for now, as the detection of parked domains is part of the DNS checks, which are part of the background processes themselves and not specifically tied to this project. The maintenance of the nameservers for parked domains will be handled there as well.
